canonical-web-and-design/ubuntu.com
# Remove plex dupe

## Done

- Removed dupliacted plex file

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex
- Check everything still works
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
